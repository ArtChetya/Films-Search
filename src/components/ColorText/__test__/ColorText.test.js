import { mount, shallow } from 'enzyme/build';
import React from 'react';
import { ColorText } from '..';

describe('ColorText component', () => {
    it('should be rendered correctly', () => {
        const component = shallow(<ColorText />);
        expect(component).toMatchSnapshot();
    });

    it('should use default font color', () => {
        const defaultFontColor = '#fff';
        const component = mount(<ColorText />);

        expect(component).toHaveStyleRule('color', defaultFontColor, {
            modifier: '&&'
        });
    });

    it('should properly add font color', () => {
        const color = '#000';
        const component = mount(<ColorText fontcolor={color} />);

        expect(component).toHaveStyleRule('color', color, {
            modifier: '&&'
        });
    });
});
