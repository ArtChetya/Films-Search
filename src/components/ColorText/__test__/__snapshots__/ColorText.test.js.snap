// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorText component should be rendered correctly 1`] = `ShallowWrapper {}`;
